src/main/java/org/apache/commons/compress/archivers/sevenz/SevenZOutputFile.java

```diff
@@ -128,6 +130,15 @@ private static <T> Iterable<T> reverse(final Iterable<T> i) {
 
     private AES256Options aes256Options;
 
+    private long entryPackedSize = 0;
+
+    /**
+     * This method returns the packedSize of the entry.
+     */
```

Comment:
All new public and protected methods need a Javadoc since tag.

Reply:
Do I have to write a more detailed information?